### DISCUSSION TITLE
Make scikit-build output less verbose

### DISCUSSION THREAD

**Discussion by MatthiasKohl** [scikit-build/scikit-build#767] [Q&A] [answered]
The output of scikit-build is very verbose since it outputs a line for every file, which can be a lot for projects with many dependencies.
This is regardless of whether we actually "install" things or not, so this will happen with `python setup.py build_ext --inplace` as well.

Is it possible to tell scikit-build not to output this kind of info?

Any lines starting with `-- Installing:` / `-- Up-to-date:` for cmake/ninja and `copying` / `writing` for cython extensions ?

**Comment by henryiii** [comment, marked as answer]
We might be able to clean up a bit (I'd really like to get the order right, at least!) before the rewrite is ready, but clean output will be a heavy emphasis of scikit-build-core.

**Reply by MatthiasKohl** [reply]
Great, that sounds cool ! I'll be waiting for the rewrite then :)
